projects/igniteui-angular/src/lib/core/deprecateDecorators.ts

```diff
@@ -1,108 +1,5 @@
 import { isDevMode } from '@angular/core';
 
```

Comment:
This file becomes pointless with just `showMessage`  in it. The function itself is also only used in only 2 places, one being a deprecation that's 2 years old (you can just remove the warning I guess) . And another which just shows a warning that can be implemented in another way.  Even if you choose to keep `showMessage` move it to some util file and remove this file entirely.